crates/tabby-scheduler/src/lib.rs

```diff
@@ -55,32 +55,3 @@ pub async fn scheduler(now: bool) -> Result<()> {
 
     Ok(())
 }
-
-#[cfg(test)]
```

Comment:
Could you move this to integration test: https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html

and only run unit test in ci?